app/models/grda_warehouse/hmis_export.rb

```diff
@@ -110,11 +112,15 @@ def filter
       ::Filters::HmisExport.new(options)
     end
 
+    def export_file_name
+      "HMIS_export_#{created_at.to_s.delete(',')}.zip"
```

Comment:
we combine underscores and spaces here, maybe one or the other?

```suggestion
      "HMIS_export_#{created_at.to_s.gsub(/\W+/, '_')}.zip"
```

or
```suggestion
      "HMIS export #{created_at.to_s.delete(',')}.zip"
```

Reply:
Good call